newIDE/app/src/ResourcesList/ResourceSource.js

```diff
@@ -129,6 +129,7 @@ export type ChooseResourceProps = {|
   ) => void,
   options: ChooseResourceOptions,
   automaticallyOpenIfPossible?: boolean,
+  resourcesImporationBehavior: string,
```

Comment:
You can use the same type as in the preferences here. You could even create a type `ResourceImportationBehavior`